app/forms/mobius_work_form.rb

```diff
@@ -5,11 +5,13 @@
 #
 # @see https://github.com/samvera/hyrax/wiki/Hyrax-Valkyrie-Usage-Guide#forms
 # @see https://github.com/samvera/valkyrie/wiki/ChangeSets-and-Dirty-Tracking
+#  updated to only work for flexible true since app is now using flexible metadata
```

Comment:
Not a deal breaker since it's just a comment, but ideally this would have been removed when the flexible code was reinstated.

Reply:
Along with the similar comments in the other files.

Reply:
Thanks for the catch. I'll remove them.